primers.go

```diff
@@ -103,3 +104,80 @@ func MeltingTemp(sequence string) float64 {
 	meltingTemp, _, _ := SantaLucia(sequence, primerConcentration, saltConcentration, magnesiumConcentration)
 	return meltingTemp
 }
+
+/******************************************************************************
+Start of the De Bruijn stuff
+******************************************************************************/
+
```

Comment:
Give biocontext and why this is useful enough to be public.

Reply:
Will do with biocontext.

The single letter variables are there because I don't really know what it is doing either (I just took examples from RosettaCode). I should figure out that portion myself.